pages/common/vimdiff.md

```diff
@@ -0,0 +1,24 @@
+# vimdiff
+
+> Open up to four files in Vim and show their differences.
+> See Vim documentation for how to edit and navigate within a window.
+
+- Open files and show the differences:
+
+`vimdiff {{file1}} {{file2}}`
+
+- Use horizontal window split instead of the default vertical split:
+
+`vimdiff -o {{file1}} {{file2}}`
+
+- Close current window without saving changes:
```

Comment:
Yea, the moving between windows is fine. For the quit and save part, that's why I mentioned to put the word "vim" in backticks in the heading section. The node-client will automatically read that and show suggestions for users to try a `tldr vim` too. 

I would suggest you change the description of "for how to edit and navigate within a window" and make it generic enough to include the functionality of `:q!` and `:w`. Therefore, users will do a `tldr vim` when needed.